pycds/orm/tables.py

```diff
@@ -174,6 +248,48 @@ class History(Base):
 Index("meta_history_freq_idx", History.freq)
 
 
+class HistoryHistory(Base):
+    """This class maps to the table which represents a history record for
+    a weather station. Since a station can potentially (and do) move
+    small distances (e.g. from one end of the airport runway to
+    another) or change the frequency of its observations, this table
+    records the details of those changes.
+
+    WARNING: The GeoAlchemy2 `Geometry` column (attribute `the_geom`) forces
+    all reads on that column to be wrapped with Postgis function `ST_AsEWKB`.
+    This may or may not be desirable for all use cases, specifically views.
+    See the GeoAlchemy2 documentation for details.
+    """
+
+    __tablename__ = hx_table_name(History.__tablename__, schema=None)
+
+    # Columns
+    history_id = Column(Integer, nullable=False, index=True)
+    station_id = Column("station_id", Integer)
```

Comment:
Should at least FKs also be not nullable?

Reply:
Ugh, yes, good catch. And it should be so in the primary table from which this was copied. And generally speaking so in all relationships (with one or two possible exceptions). I'll create an issue for that.

Reply:
And ... since the column is actually _copied_ directly from the base table in the migration CREATE TABLE directive, I can't (or shouldn't) do this until the base table is updated. See issue #230 .